packages/@react-spectrum/table/docs/TableView.mdx

```diff
@@ -85,43 +85,40 @@ which dramatically improves performance. Make sure that each rendered property i
 property within each Row object.
 
 ```tsx example
-function Example() {
-  let columns = [
-    {name: 'Name', uid: 'name'},
-    {name: 'Type', uid: 'type'},
-    {name: 'Date Modified', uid: 'date'}
-  ];
-
-  let rows = [
-    {id: '1', name: 'Games', date: '6/7/2020', type: 'File folder'},
-    {id: '2', name: 'Program Files', date: '4/7/2021', type: 'File folder'},
-    {id: '3', name: 'bootmgr', date: '11/20/2010', type: 'System file'},
-    {id: '4', name: 'log.txt', date: '1/18/2016', type: 'Text Document'}
-  ];
-
-  return (
-    <Flex flexGrow={1} maxWidth="size-6000">
```

Comment:
Do we wanna remove the Flex wrapper from the rest of the examples for consistency?

Reply:
yeah I think Danni hasn't pushed her changes yet.